MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+# Kubernetes Project Maintainership
+
+Kubernetes is a big project with many contributors, the project organization is
+a daunting task and the project maintainers are the members that are trusted
+with that task, driving the project, improving the health of the project,
+performing quality control and being responsible over one or more components in
+the project. A Kubernetes maintainer is a contributor who has contributed/is
+contributing to the project regularly and a developer that demonstrated good
+technical judgement. The following is a list of requirements to be a Kubernetes
+maintainer.
+
+Before continuing, it should be noted that we value every contributions, however
+small it may be, even a comment on an issue a valuable contribution to us.
+Maintainership requires time commitment and community trust. You don't have to
+be a maintainer to have a meaningful impact on Kubernetes!
+
+## How to become a maintainer
+
+### 1. Be an active contributor
+
+An active contributor is someone who contributed regularly in the last 2 months
+and is one of the top 20 of contributors for the last 2 months.
+
+For example in the context of merged commit contributions, from September to
+November, the 20th developer has 20 commits.
```

Comment:
> I don't think metrics like commit/PR counts are a good measure of this. I worry that tying those to eligibility for maintainership will lead to gaming.

Seconded.  Commits are very vulnerable to gaming and not concretely related to maintenance or new feature fulfillment.  They commonly correlate, but not strongly enough to make decisions like this about.

Take https://github.com/kubernetes/kubernetes/graphs/contributors?from=2016-09-01&to=2016-11-01&type=c as an example.  Missing from the top20, but probably maintainers bgrant, thockin, lavalamp, derekwaynecarr, and pmorie.  (Also, in spite of the example, mine is not gamed.  It merely points to successive refactors while maintaining a working system.  Turns out its hard to there from here...)